cirq/contrib/noise_models/noise_models_test.py

```diff
@@ -315,14 +316,109 @@ def test_aggregate_decay_noise_after_moment():
         values: [{
             double_val: .003
         }]
+    }, {
+        name: 'single_qubit_readout_separation_error',
+        targets: ['q0_0'],
+        values: [{
+            double_val: .004
+        }]
+    }, {
+        name: 'single_qubit_readout_separation_error',
+        targets: ['q0_1'],
+        values: [{
+            double_val: .005
+        }]
+    }, {
+        name: 'single_qubit_readout_separation_error',
+        targets: ['q1_0'],
+        values: [{
+            double_val: .006
+        }]
+    }, {
+        name: 'single_qubit_idle_t1_micros',
+        targets: ['q0_0'],
+        values: [{
+            double_val: .007
+        }]
+    }, {
+        name: 'single_qubit_idle_t1_micros',
+        targets: ['q0_1'],
+        values: [{
+            double_val: .008
+        }]
+    }, {
+        name: 'single_qubit_idle_t1_micros',
+        targets: ['q1_0'],
+        values: [{
+            double_val: .009
+        }]
     }]
 """, v2.metrics_pb2.MetricsSnapshot())
 
 
-def test_per_qubit_noise_from_data():
-    # Generate the noise model from calibration data.
+def test_noise_from_metrics_requires_type():
+    # Attempt to generate a noise model without specifying a noise type.
+    calibration = cirq.google.Calibration(_CALIBRATION_DATA)
+    with pytest.raises(ValueError) as e:
```

Comment:
there is a helper for matching error messages.  
``
with pytest.raises(ValueError, match='regext matching string'):

Reply:
Done.